### ISSUE TITLE
I don't want default content

### ISSUE THREAD

**Issue by BWPanda** [backdrop/backdrop-issues#5587]
## Description of the need

I want to setup a new site and either:
1. Not have default content, content types, vocabularies, etc. created by default, or
2. Be able to remove the default content, content types, vocabularies, etc. with the press of a button.

## Proposed solution

Either we:
1. Allow users to select the minimal profile on install (I.e. we unhide it and show the profile selection step of the installer (currently its skipped)), or
2. Move default content, content types, vocabularies, etc. to a dedicated module that can be disabled as needed.

Option 1 would be similar to/in favour of https://github.com/backdrop/backdrop-issues/issues/2000 (since choosing between Standard and Minimal, is essentially the same as making Minimal the default and then choosing to install Standard over the top or not), but would go against https://github.com/backdrop/backdrop-issues/issues/1999 and https://github.com/backdrop/backdrop-issues/issues/467.

Option 2 would allow new users to see the default content, etc., but would also give them an option to remove it should they later choose to start afresh. However, it's going to be tricky to work out how to handle non-content things (e.g. content types, vocabularies, etc.).

## Alternatives that have been considered

Users can already choose to install Backdrop without any default content, etc. by using the hidden Minimal install profile, but you have to already know about it, and it's not very easy to do. Also, it means giving up on lots of other stuff that the Standard profile does other than default content.

Users can also just manually delete what they don't want, but in addition to what core already provides, we're about to add even more default content, layouts, views, etc. with https://github.com/backdrop/backdrop-issues/issues/4903.

## Additional information

I don't know which of the proposed solutions is the best, I just know that I think users should have a clear choice about what goes on their site by default.

**Comment by indigoxela** [comment]
> I don't want default content

Hehe, that's a statement. :grinning:

Depending on what the project will be, the content types, vocab, roles, enabled modules... may fit or may not fit. In almost all cases they all need tweaks. So sometimes the ability to decide for an (empty) profile on install is a missing feature (compared to Drupal).

But I also get the intention behind the effort to make it a "super easy install with everything in place without any decision". I find the Backdrop install pretty cool an quick. And after install you have a ready-to-use website (unlike Drupal).

Yes, the minimal profile can be used (if one knows how), but that's a really hidden feature. So it's currently a little tricky to get a truly vanilla Backdrop without any clutter. I'm assuming that enabling the profile decision on install again, will get a lot of opposition from the people who removed it.

> I think users should have a clear choice about what goes on their site by default.

That's a really valid point, but on the other hand people who are new to Backdrop should be able to get something useful out of the box. I'm not sure which of these should have precedence.

**Comment by BWPanda** [comment]
> Hehe, that's a statement. 😀 

Yeah, I couldn't work out how to title this issue, since I'm not sure which way it'll ultimately go: un-hide the Minimal install profile, or allow default content to be deleted easily...

> That's a really valid point, but on the other hand people who are new to Backdrop should be able to get something useful out of the box. I'm not sure which of these should have precedence.

Neither. That's why this should be an option during install, then both bases are covered. But the Standard install profile should continue to be the default option, for backwards-compatibility and since it's likely closer to the 80% use-case.
